### ISSUE TITLE
Speed comparison between other methods?

### ISSUE THREAD

**Issue by quinton-ashley** [NotNinja/convert-svg#37]
It'd be helpful if there was a speed comparison between this script and other options like using inkscape from the command line.

**Comment by neocotic** [comment]
@quinton-ashley Speed has been a priority for me, which is why I supported the ability to create reusable converter instances to share Chromium instances from the beginning. That said; my primary goal was to create a quick and easy method for converting SVGs to other formats using the latest standards. I used to use `svg2png`, which inspired this project, however, I grew frustrated at the poor rendering and inconsistencies with the latest specs since it used the outdated PhantomJS. My goal was not defeat similar libraries/tools in speed.

I doubt very much that spinning up headless Chromium instances, interacting with them using their test API, and then capturing screenshots in the target formats is the optimal solution, but it is a simple one that requires little code to implement while offering the latest SVG rendering.

If you're interesting in capturing those benchmarks, I'd be interested in the results, but I doubt they'll put these libraries on top, so it'd feel weird to then publish them on the README etc.
